src/lib.rs

```diff
@@ -356,6 +360,30 @@ impl FixedBitSet
     }
 }
 
+#[cfg(feature = "std")]
+impl Binary for FixedBitSet {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), Error> {
+        let mut display_data = if f.alternate() {
+            String::from("0b")
+        } else {
+            String::new()
+        };
+
+        for block in self.data.iter().rev() {
+            display_data = display_data.add(&format!("{:b}", block));
```

Comment:
nice. I'd suggest that we just write this piece by piece instead of creating an intermediate String.